python_modules/dagster/dagster/core/instance/config.py

```diff
@@ -177,4 +253,5 @@ def dagster_instance_config_schema():
         "code_servers": Field(
             {"local_startup_timeout": Field(int, is_required=False)}, is_required=False
         ),
+        "tick_retention": tick_retention_config_schema(),
```

Comment:
one thing that we are going to want to figure out at some point - every time we add one of these fields and somebody uses it, they have to upgrade their user code as well, or it will error out when it tries to load the instance when launching a run and this new unexpected key is there (even though a setting like this is totally irrelevant in user code)